openapi.yaml

```diff
@@ -26564,6 +27640,138 @@ components:
           description: The storage type of this Volume.
           example: nvme
           readOnly: true
+    VPC:
+      type: object
+      description: An object describing a VPC belonging to the Account.
+      properties:
+        id:
+          type: integer
+          readOnly: true
+          x-linode-filterable: true
+          description: The unique ID of the VPC.
+          example: 123
+          x-linode-cli-display: 1
+        label:
+          type: string
+          minLength: 1
+          maxLength: 64
+          x-linode-filterable: true
+          description: |
+            The VPC's label, for display purposes only.
+
+            * Must be unique among the Account's VPCs.
+            * Must contain only ASCII letters, numbers, and hyphens (`-`).
+          example: cool-vpc
+          x-linode-cli-display: 2
+        description:
+          type: string
+          maxLength: 255
+          default: ""
+          description: A written description to help distinguish the VPC.
+          example: "A description of my VPC."
+          x-linode-cli-display: 3
+        region:
+          type: string
+          x-linode-filterable: true
+          description: The Region for the VPC.
+          example: us-east
+          x-linode-cli-display: 4
+        subnets:
+          type: array
+          description: A list of subnets associated with the VPC.
+          items:
+            $ref: '#/components/schemas/VPCSubnet'
+        created:
+          type: string
+          format: date-time
+          readOnly: true
+          x-linode-filterable: true
+          description: The date-time of VPC creation.
+          example: '2023-07-11T00:00:00'
+        updated:
+          type: string
+          format: date-time
+          readOnly: true
+          nullable: true
+          x-linode-filterable: true
+          description: The date-time of the most recent VPC update.
+          example: '2023-09-11T00:00:00'
+    VPCSubnet:
+      type: object
+      description: An object describing a VPC Subnet.
+      properties:
+        id:
+          type: integer
+          readOnly: true
+          x-linode-filterable: true
+          description: The unique ID of the VPC Subnet.
+          example: 456
+          x-linode-cli-display: 1
+        label:
+          type: string
+          minLength: 1
+          maxLength: 64
+          x-linode-filterable: true
+          description: |
+            The VPC Subnet's label, for display purposes only.
+
+            * Must be unique among the VPC's Subnets.
+            * Must contain only ASCII letters, numbers, and hyphens (`-`).
+          example: cool-vpc-subnet
+          x-linode-cli-display: 2
+        ipv4:
+          type: string
+          format: ip
+          nullable: true
+          description: |
+            IPv4 range in CIDR canonical form.
+
+            * A VPC Subnet requires either `ipv4`, `ipv6`, or both.
```

Comment:
If we omit all the IPv6 mentions from the BETA docs, then also remove this. For the BETA release the ipv4 property must be set (when POSTing).